src/Sentry/Internal/AotHelper.cs

```diff
@@ -13,16 +13,15 @@ static AotHelper()
         IsTrimmed = CheckIsTrimmed();
     }
 
-
     [UnconditionalSuppressMessage("Trimming", "IL2026: RequiresUnreferencedCode", Justification = AvoidAtRuntime)]
     private static bool CheckIsTrimmed()
     {
-        if (TryGetBoolean("publishtrimmed", out var trimmed))
+        if (BuildProperties.TryGetBoolean("PublishTrimmed", out var trimmed))
         {
             return trimmed;
         }
 
-        if (TryGetBoolean("publishaot", out var aot))
+        if (BuildProperties.TryGetBoolean("PublishAot", out var aot))
```

Comment:
I think this is a major bug that was introduced by the initial PR including the source generator...

Reply:
Fixed in #4333:

Changed the generated code to use case-insensitive lookup:
```diff
-BuildProperties.Initialize(new Dictionary<string, string>()
+BuildProperties.Initialize(new Dictionary<string, string>(StringComparer.OrdinalIgnoreCase)0
```